lib/src/list_notifier.dart

```diff
@@ -5,12 +5,23 @@ import 'package:flutter/foundation.dart' show ChangeNotifier, ValueListenable;
 import 'package:functional_listener/functional_listener.dart';
 
 /// A List that behaves like `ValueNotifier` if its data changes.
-/// it does not compare the elements on bulk operations
-/// If you set [notifyIfEqual] to `false` it will compare if a value passed value
-///  passed is equal to the existing value.
-/// like `list[5]=4` if the content at index 4 is equal to 4 and only call
+/// It does not compare the elements on bulk operations
+///
+/// If you set [notificationMode] to [normal] `ListNotifier` will compare if a
+/// value passed is equal to the existing value.
+/// Like `list[5]=4` if the content at index 4 is equal to 4 and only call
```

Comment:
Like in `list[5]=4` it tests if the content at index 4 is equal to 4 and only call